File: app/src/main/java/github/umer0586/util/UIUtil.java
package github.umer0586.util;

import android.os.Handler;
import android.os.Looper;

public class UIUtil {

    private static Handler handler = new Handler(Looper.getMainLooper());

    public static void runOnUiThread(Runnable runnable)
    {
        handler.post(runnable);
    }

}

File: app/src/main/java/github/umer0586/sensorserver/util/UIUtil.kt
package github.umer0586.sensorserver.util

import android.os.Handler
import android.os.Looper

object UIUtil
{


    private val handler = Handler(Looper.getMainLooper())
    fun runOnUiThread(runnable: Runnable?)
    {
        handler.post(runnable!!)
    }
}